lib/utils/package-manager.js

```diff
@@ -71,7 +72,31 @@ function getPackageManager() {
 	return "yarn";
 }
 
+/**
+ *
+ * Returns the path to globally installed
+ * npm packages, depending on the available
+ * package manager determined by `getPackageManager`
+ *
+ * @returns {String} path - Path to global node_modules folder
+ */
+function getPathToGlobalPackages() {
```

Comment:
Is this really needed? We already have a package manager function that takes care of this?

Reply:
Which function are you referring to? I don't see any code in `pacakge-manager.js` that serves the same purpose.